packaging/fedora/charliecloud.spec

```diff
@@ -16,11 +16,14 @@ Summary:       Lightweight user-defined software stacks for high-performance com
 License:       ASL 2.0
 URL:           https://hpc.github.io/%{name}/
 Source0:       https://github.com/hpc/%{name}/releases/downloads/v%{version}/%{name}-%{version}.tar.gz
-BuildRequires: gcc rsync /usr/bin/python3
-Patch0:        lib64.patch
-%if 0%{?el7}
+BuildRequires: gcc rsync bash
+Requires:      squashfuse squashfs-tools
 Patch1:        el7-pkgdir.patch
-%endif
+# Suggests:    name-builder docker buildah
+Provides:      %{name}-runtime
+Obsoletes:     %{name}-runtime < 0.24-9
+Provides:      %{name}-common
```

Comment:
Can we have just obsoletes? The `charliecloud-common` package only existed for such a brief while; seems a shame to carry around this forever. I'm also skeptical it's something we should "provide" — people should not be depending on it.

Reply:
Good questions and comments.

Re. obsoletes: we _could_ just have obsoletes but then we'd need to add a new `rpmlint` ignore entry since rpmbuild will complain that we aren't specifying versions. FWIW I'd like for these obsoletes lines to disappear completely in 0.26 since, as you pointed out, it only existed for a day or two on f35/34. So I didn't mind adding the versions with their imminent removal in the near future in mind. Thoughts?